src/plugins/vis_type_tagcloud/public/components/collections.ts

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * and the Server Side Public License, v 1; you may not use this file except in
+ * compliance with, at your election, the Elastic License or the Server Side
+ * Public License, v 1.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { TagCloudVisParams } from '../types';
+
+interface Scales {
+  text: string;
+  value: TagCloudVisParams['scale'];
+}
+
+interface Orientation {
+  text: string;
+  value: TagCloudVisParams['orientation'];
+}
+
+interface Collections {
+  scales: Scales[];
+  orientations: Orientation[];
+}
+
+export const collections: Collections = {
```

Comment:
nit: I don't have a strong opinion here but I see that in `src/plugins/vis_type_xy/public/editor/collections.ts` all collections was declared as  a simple functions.  I don't think that I want to see the same here but probably we should use one style for that.